modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,272 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-classic-aws-rhcos-ami_{context}"]
+= {product-title} AMIs 
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {product-title} nodes.
+
+[id="rosa-classic-aws-rhcos-ami-415_{context}"]
+== Version 4.15 AMIs
+
+[cols="2a,2a",options="header"]
+|===
+
+|AWS zone
+|AWS AMI
+
+|`us-east-1`
+|`ami-04af4cb679580f10d`
+|`us-east-2`
+|`ami-03780ec2f9f1a9502`
+|`us-west-1`
+|`ami-0ac973efc5712480d`
+|`us-west-2`
+|`ami-0b97fd4d1c35daf3c`
+|`af-south-1`
+|`ami-0807e6acaf49bfdb4`
+|`ap-east-1`
+|`ami-080041b23262f46df`
+|`ap-south-2`
+|`ami-056bbb876992a2b9f`
+|`ap-southeast-3`
+|`ami-0288fd72ccdbfd7a7`
+|`ap-southeast-4`
+|`ami-0390e29ac66acf928`
+|`ap-south-1`
+|`ami-09e9254ef26ac6330`
+|`ap-northeast-3`
+|`ami-02e77a92d0699aec0`
+|`ap-northeast-2`
+|`ami-0b6a0d64e49e51b1c`
+|`ap-southeast-1`
+|`ami-0c22bbd807a0256d6`
+|`ap-southeast-2`
+|`ami-06be0f08377760fc9`
+|`ap-northeast-1`
+|`ami-000fd95e97985af95`
+|`ca-central-1`
+|`ami-0a668d27ee6f2d05b`
+|`eu-central-1`
+|`ami-072a4b61d61802fc5`
+|`eu-west-1`
+|`ami-020e22cc494ab6f27`
+|`eu-west-2`
+|`ami-03c005b7cfa8114cd`
+|`eu-south-1`
+|`ami-0f42b5d7603ace8c5`
+|`eu-west-3`
+|`ami-02c056bb061ecada0`
+|`eu-south-2`
+|`ami-0b8953c88a0645080`
+|`eu-north-1`
+|`ami-099f3930c1ec7cc46`
+|`eu-central-2`
+|`ami-0060d33ef19ed919a`
+|`me-south-1`
+|`ami-08be609f9921b78a1`
+|`me-central-1`
+|`ami-0b119c2aa0ffd04c5`
+|`sa-east-1`
+|`ami-06f5a474fe634c772`
+
+|===
+
+[id="rosa-classic-aws-rhcos-ami-414_{context}"]
+== Version 4.14 AMIs
+
+[cols="2a,2a",options="header"]
+|===
+
+|AWS zone
+|AWS AMI
+
+|`us-east-1`
+|`ami-06f3dda44132a8cc9`
+|`us-east-2`
+|`ami-04737283ac19b8635`
+|`us-west-1`
+|`ami-04eda993afcf68041`
+|`us-west-2`
+|`ami-0ca469d02b829bed2`
+|`af-south-1`
+|`ami-0d401016e8a21cecb`
+|`ap-east-1`
+|`ami-0cac00e174222f885`
+|`ap-south-2`
+|`ami-0b13f44eee5d62cba`
+|`ap-southeast-3`
+|`ami-05ebb895e2d189a92`
+|`ap-southeast-4`
+|`ami-0edf2b052abc2af7d`
+|`ap-south-1`
+|`ami-01347c0f82aa8ccea`
+|`ap-northeast-3`
+|`ami-087bfb06897e9e17e`
+|`ap-northeast-2`
+|`ami-0144a4f381fc1606c`
+|`ap-southeast-1`
+|`ami-032de9543fa88f496`
+|`ap-southeast-2`
+|`ami-01c8c1a3b43c3d128`
+|`ap-northeast-1`
+|`ami-0d8e30e0eba1db1ae`
+|`ca-central-1`
+|`ami-0708cfc5575d806d7`
+|`eu-central-1`
+|`ami-06e39a701e2b23dd5`
+|`eu-west-1`
+|`ami-0eba9f75078b5554a`
+|`eu-west-2`
+|`ami-05c3f2ad7963594f3`
+|`eu-south-1`
+|`ami-037df250c060f1a85`
+|`eu-west-3`
+|`ami-05967a4e65a8356d9`
+|`eu-south-2`
+|`ami-0f3ad9b25036a95dc`
+|`eu-north-1`
+|`ami-0fb33a85b51cb99ef`
+|`eu-central-2`
+|`ami-04c6ff1edabf3f212`
+|`me-south-1`
+|`ami-02d91107af59d29a2`
+|`me-central-1`
+|`ami-08984cd1e1423187e`
+|`sa-east-1`
+|`ami-0ea74c113800ca0d2`
+
+|===
+
+[id="rosa-classic-aws-rhcos-ami-413_{context}"]
+== Version 4.13 AMIs
```

Comment:
Comparing the output of 'rosa list versions -o json|jq -r '[.[]|select(.raw_id | contains("4.13"))]'|jq -r '.[0].image_overrides.aws[]| "(.region.id):(.ami)"''. This document PR misses bellow region ami:
ap-southeast-4:ami-00e8efb88384ef992
ca-west-1:ami-0ef0783b0847bf4d4
us-gov-east-1:ami-032a0e0ea6912d363
us-gov-west-1:ami-0d4c03b0bf5595597

And other ami ids are different between them.